### ISSUE TITLE
Proper syntax highlighting

### ISSUE THREAD

**Issue by dkundel** [dominictarr/tslide#15]
Hey everyone,

just one idea. Have you considered adding support for something like [mikaelbr/marked-terminal](https://github.com/mikaelbr/marked-terminal)? This way you still have JS syntax highlighting but also colors for other things such as headlines.

Cheers,
Dominik

**Comment by rafaelrinaldi** [comment]
Hey @dkundel, thanks for the suggestion. In my opinion tslide already does a good job at that so adding a new library to the table just to do some minor tweaks is not worth the effort. Let's hear what others say 👍

**Comment by hackergrrl** [comment]
Note that this will change presentation beyond just highlighting: `tslide` currently applies some styling to markdown (like boldening headings), but it doesn't _render_ the markdown. e.g. `# foo` doesn't become **foo**; the markdown formatting remains. Is changing this desirable?

**Comment by rafaelrinaldi** [comment]
@noffle Yeah, that's true. It's not something I expect at least. Happy with the current behavior.

**Comment by dominictarr** [comment]
I think he means the highlighting in code blocks, that could certainly be improved

**Comment by dkundel** [comment]
Well this library would on the one hand fix the code highlighting. `marked-terminal` doesn't change the markdown itself either. Like it doesn't make headlines bigger. But it highlights them using different colors, supports inline boldness like `tslide` already has, supports markdown tables and others and has various other configurations. 

This is the output of the example that they have in their code:

<img width="838" alt="screen shot 2016-09-07 at 11 07 53 pm" src="https://cloud.githubusercontent.com/assets/1505101/18328691/34690aaa-7550-11e6-9a54-1bb00c0d7f7a.png">
<img width="836" alt="screen shot 2016-09-07 at 11 08 01 pm" src="https://cloud.githubusercontent.com/assets/1505101/18328690/34673a68-7550-11e6-9008-103c3d38a8d9.png">

**Comment by hackergrrl** [comment]
Awesome! My own darn fault for not actually reading what the module does. As long as the formatting isn't being changed, +100 to adding this. Maybe bonus points if there's a flag to disable, for the old guard.

**Comment by rafaelrinaldi** [comment]
@dominictarr As far as I understood he wants to introduce highlighting for markdown, not only code blocks. Right now the only thing we do is basic stuff like bold/italic, this would introduce colors and whatnot.

It can be an interesting feature but not something that I personally miss. If you guys are OK with it, I am too :metal:

**Comment by dominictarr** [comment]
worth it to get proper code highlighting, I think. gonna merge

**Comment by dominictarr** [comment]
and then I realized it's not a PR yet! @dkundel would you like to make one?

**Comment by dkundel** [comment]
Yeah I'll work on it today or tomorrow and send a PR!

**Comment by hackergrrl** [comment]
Very excited now for this feature to land! :star2:
